src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseData.java

```diff
@@ -204,7 +205,7 @@ public class SscsCaseData implements CaseData {
     private String adjournCaseAreDirectionsBeingMadeToParties;
     private String adjournCaseDirectionsDueDateDaysOffset;
     private String adjournCaseDirectionsDueDate;
-    private String adjournCaseTypeOfNextHearing;
+    private HearingChannel adjournCaseTypeOfNextHearing;
```

Comment:
You can't just change this type, it's already in use in different parts of the system. You need to resolve the hearing channel from the string